stories/Collapse/CollapseExample.js

```diff
@@ -28,4 +28,6 @@ const CollapseExample = () => {
   )
 }
 
-export default CollapseExample
+const CollapseExampleWithHook = props => <CollapseExample {...props} />
+
+export default CollapseExampleWithHook
```

Comment:
This is not required. Can you revert it?
It breaks the `Source Code` block in the addon-info panel.